src/test/kotlin/com/terraformation/backend/tracking/edit/PlantingSiteEditCalculatorTest.kt

```diff
@@ -829,6 +823,44 @@ class PlantingSiteEditCalculatorTest {
         desired)
   }
 
+  @Test
+  fun `updates existing subzone if original zone is deleted and new zone has a subzone with its name`() {
+    val existing =
+        existingSite(width = 1000) {
+          zone(name = "A", width = 500, numPermanent = 1) { subzone(name = "1") { cluster() } }
+          zone(name = "B", width = 500, numPermanent = 1) { subzone(name = "2") { cluster() } }
```

Comment:
I think this test will need to be updated once the other cluster PRs are merged, or vice versa (because of the use of `cluster()`